### PR TITLE
User story 10

### PR DESCRIPTION
**Pull request by EricLarson2020**
Created Register user functionality and corresponding tests

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+
+    user = User.new(user_params)
```
Comment (EricLarson2020) [inline app/controllers/users_controller.rb:9]: if user is saved then a message is sent to the profile page with their name

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  validates :email, uniqueness: true, presence: true
+  validates :password, :presence =>true, :confirmation =>true
```
Comment (whitneykidd) [inline app/models/user.rb:3]: Nice use of password confirmation here

File: spec/features/user/index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    fill_in :confirm_password, with: "1234"
+    click_button "Create User"
+    expect(current_path).to eql("/profile")
+    expect(page).to have_content("You are now logged in Bob")
+  end
+end
+
+
+
+  # jack = User.new ({
```
Comment (EricLarson2020) [inline spec/features/user/index_spec.rb:26]: Clean this up

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  validates :email, uniqueness: true, presence: true
```
Comment (EricLarson2020) [inline app/models/user.rb:2]: research need for presence:true

Comment (jhgould) [reply app/models/user.rb:2]: Good call. Lets circle back once we have some time.

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (jhgould) [reply app/controllers/users_controller.rb:9]: Will we need to use sessions for future user stories?

**Pull request merged**